tools/querytee/proxy_metrics.go

```diff
@@ -43,6 +43,8 @@ func NewProxyMetrics(registerer prometheus.Registerer) *ProxyMetrics {
 			Name:      "responses_compared_total",
 			Help:      "Total number of responses compared per route name by result.",
 		}, []string{"route", "result"}),
+		/// add in metrics for latency
```

Comment:
is this a TODO?

Reply:
whoops, yes, this shouldn't have been commited. Thanks.